From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Tue, 24 Jan 2017 19:59:52 +0100
Subject: [PATCH] f_odlpluginbuild: Always inherit Armband settings

This should eliminate any possible confusion when building the
ODL plugin manually (outside OPNFV ISO build), which used to
require setting up some additional enviroment variables, like
UBUNTU_ARCH.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 build/f_isoroot/f_odlpluginbuild/Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/build/f_isoroot/f_odlpluginbuild/Makefile b/build/f_isoroot/f_odlpluginbuild/Makefile
index d3a7ea8..4c77abc 100644
--- a/build/f_isoroot/f_odlpluginbuild/Makefile
+++ b/build/f_isoroot/f_odlpluginbuild/Makefile
@@ -9,6 +9,11 @@

 TOP := $(shell pwd)

+# Configure Armband specific stuff when ARMBAND_BASE is set
+ifdef ARMBAND_BASE
+    include ${ARMBAND_BASE}/armband-fuel-config.mk
+endif
+
 include config.mk

 .PHONY: all
